modules/python/src2/gen2.py

```diff
@@ -768,8 +776,16 @@ def gen_code(self, codegen):
             # if the function/method has only 1 signature, then just put it
             code += all_code_variants[0]
         else:
-            # try to execute each signature
-            code += "    PyErr_Clear();\n\n".join(["    {\n" + v + "    }\n" for v in all_code_variants])
+            # try to execute each signature, add an interlude between function 
+            # calls to collect error from all conversions
+            code += '    conversionErrors.clear();\n' \
+                    '    conversionErrors.reserve({});\n'.format(len(all_code_variants))
+            code += '    \n'.join(gen_template_overloaded_function_call.substitute(variant=v)
+                                  for v in all_code_variants)
+            code += '    if (!conversionErrors.empty())\n' \
```

Comment:
Could this condition be `false`? (`conversionErrors.empty()` is true)

Reply:
In normal scenario: no - every overload resolution should succeed or fail setting an error, but there might be ill-formed `pyopencv_to` functions e.g. `pyopencv_to_generic_vec` - which has [code path](https://github.com/opencv/opencv/blob/69b57eb1190338397eb28841d10eb4a7e933b99a/modules/python/src2/cv2.cpp#L1555-L1556) returning false, but without setting an error moreover there are some generated conversion (still need to be checked).
I think we can leave this check until stated problem is resolved (all conversions are tested - there not so many left, because we need to implement correct parsing objects those conform to `sequence protocol` e.g. tuples, lists, 1d numpy arrays or user classes (will be supported automatically)). What do you think?